firebase.json

```diff
@@ -1,41 +1,19 @@
 {
   "hosting": {
     "public": "build",
-    "headers": [
-      {
-        "source": "apple-app-site-association",
-        "headers": [
-          {
-            "key": "Content-Type",
-            "value": "application/json"
-          }
-        ]
-      }
-    ],
-    "appAssociation": "NONE",
```

Comment:
Why did you removed this? This is extremely important for the apps' deep linking.